lib/_tls_wrap.js

```diff
@@ -141,29 +141,23 @@ function onclienthello(hello) {
     if (err)
       return self.destroy(err);
 
-    // Servername came from SSL session
-    // NOTE: TLS Session ticket doesn't include servername information
-    //
-    // Another note, From RFC3546:
-    //
-    //   If, on the other hand, the older
-    //   session is resumed, then the server MUST ignore extensions appearing
-    //   in the client hello, and send a server hello containing no
-    //   extensions; in this case the extension functionality negotiated
-    //   during the original session initiation is applied to the resumed
-    //   session.
-    //
-    // Therefore we should account session loading when dealing with servername
-    var servername = session && session.servername || hello.servername;
-    loadSNI(self, servername, function(err, ctx) {
+    self._handle.endParser();
+  });
+}
+
+
+function oncertcb(info) {
+  var self = this;
+  var servername = info.servername;
+
+  loadSNI(self, servername, function(err, ctx) {
+    if (err)
+      return self.destroy(err);
+    requestOCSP(self, info, ctx, function(err) {
```

Comment:
`ctx` is `undefined` if (!servername || !self._SNICallback) in loadSNI. Do we require SNI to use OCSP stapling?